common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -141,6 +141,17 @@ public static boolean isHomeTenantAlias(@NonNull final String tenantId) {
                 || tenantId.equalsIgnoreCase(ORGANIZATIONS);
     }
 
+
+    /**
+     * Util method which returns true if the tenant alias is "common" ,
+     * "consumers" or MSA mega tenant.
+     */
+    public static boolean isMsaHomeTenantAlias(@NonNull final String tenantId) {
+        return tenantId.equalsIgnoreCase(AzureActiveDirectoryAudience.ALL)
```

Comment:
technically "common" does not necessarily translate to "MSA home tenant".

Reply:
I will use the logic directly at caller site to open it as public method.